content/coronavirus_landing_page.yml

```diff
@@ -3,14 +3,16 @@ content:
   meta_description: "Find information on coronavirus, including guidance, support, announcements and statistics."
   page_header: "Coronavirus (COVID&#8209;19)"
   stay_at_home:
-    pretext: Stay at home
+    pretext: Stay alert
     list:
-      - Only go outside for food, health reasons or work (but only if you cannot work from home)
-      - If you go out, stay 2 metres (6ft) away from other people at all times
-      - Wash your hands as soon as you get home
+      - stay at home as much as possible
+      - work from home if you can
+      - limit contact with other people
+      - keep your distance if you go out (2 metres apart where possible)
+      - wash your hands regularly
   guidance:
-    pretext-1: Do not meet others, even friends or family.
-    pretext-2: You can spread the virus even if you don’t have symptoms.
+    pretext-1: "We can all help control the virus if we all stay alert. This means you must:"
```

Comment:
We might want to try to re-design this so that it comes directly before the list. It feels confusing here. But not for now!

Reply:
Yeah agree, I got really confused doing it. Will do tomorrow!